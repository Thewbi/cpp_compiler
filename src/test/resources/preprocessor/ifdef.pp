#define DEBUG

int main() {

  #ifdef DEBUG
    printf("Debug mode is ON\n");
  #endif

  return 0;
}