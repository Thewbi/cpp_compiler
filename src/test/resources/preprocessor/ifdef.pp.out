int main sub ( ) { 
   return 0 ; 
} 
